pkg/filters/filters_test.go

```diff
@@ -185,3 +186,16 @@ func TestComputeUntilTimestamp(t *testing.T) {
 		})
 	}
 }
+
+func TestGetLibpodFilters(t *testing.T) {
+	testFilters := map[string][]string{
+		"label":     {"foo=bar", "xyz"},
+		"reference": {"image-name"},
+	}
+	expectedLibpodFilters := []string{"label=foo=bar", "label=xyz", "reference=image-name"}
+	t.Run("Testing getLibpodFilters", func(t *testing.T) {
```

Comment:
(The `t.Run` here does ~nothing, when there are no other test cases in the function. It also doesn’t hurt. …)

… just if this function remains structured for multiple tests, the test input and output probably belongs inside this test.

Reply:
Fixed